src/services/WalletService/WalletService.js

```diff
@@ -1,3 +1,78 @@
-export default class WalletService {
+/* eslint-disable no-console */
+/* eslint-disable class-methods-use-this */
+import WorkerWrapper from './entities/WorkerWrapper';
+import WalletWorker from '../../workers/wallet.worker';
+import { fs, path, PATH_TO_WALLETS } from '../../constants';
 
+
+const bip39 = require('bip39');
+/** Service for working with wallets */
+
+const worker = new WorkerWrapper(new WalletWorker());
+class WalletService {
+  /**
+   * Decrypts wallet
+   * @param {string} url path to wallet
+   * @param {string} password password for decrypting
+   * @returns {object} Wallet instance
+   */
+  readWallet(input, password) {
+    const data = {
+      action: 'read',
+      input,
+      password,
+    };
+    return worker.send(data);
+  }
+
+  /**
+   * Write encrypted wallet to file
+   * @param {string} encryptedWallet
+   * @return {bool} write status: 1 - success, 2 - error
+   */
+  writeWalletToFile(encryptedWallet) {
+    let date = new Date();
+    date = `${date.getUTCFullYear()}-${date.getUTCDate()}-${date.getUTCDay()}T${date.getUTCHours()}-${date.getUTCMinutes()}-${date.getUTCSeconds()}.${date.getMilliseconds() * 1000000}Z`;
```

Comment:
try to use some library names to get wallet name, it looks awfull

Reply:
I cant find any library for naming wallets like geth, so this will stay like that was